jolly-jellyfish/src/page_maker/templates/page_maker/webpage_detail.html

```diff
@@ -0,0 +1,22 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<div><h1>{{ webpage.user_title }}</h1></div>
+<div>
+  {% for comment in comments %}
+  <div>
+    <p>{{ comment.author.username }}: {{ comment.title }}</p>
+    <p>{{ comment.content }}</p>
+  </div>
```

Comment:
`table`s shouldn't be used for non-table layout, but I will certainly add some styling to comments - it would help if these `p`s and `div`s had classes.